station.localization.json

```diff
@@ -2924,47 +2924,47 @@
       "ident_ios": "",
       "ident_android": "settings_chart_interval_description",
       "en": "Set interval between points for chart from 1 to 60 minutes. Less value means more accurate chart, but higher requirements to performance of your device",
-      "fi": "",
+      "fi": "Aseta historian tallennusväli 1 - 60 minuuttia. Tiheämpi tallennusväli mahdollistaa tarkempien historiatietojen esittämisen, mutta tämä käyttää enemmän laitteen resursseja",
       "ru": "Установите интервал между точками на графиках от 1 до 60 минут. Чем меньше значение, тем выше точность отображения. Но в тоже время повышаются требования к быстродействию вашего телефона.",
       "sv": ""
     },
     {
       "ident_ios": "",
       "ident_android": "settings_chart_view_period",
       "en": "Chart history view period",
-      "fi": "",
+      "fi": "Historian tarkastelujakso",
       "ru": "Диапазон отображения графиков",
       "sv": ""
     },
     {
       "ident_ios": "",
       "ident_android": "settings_chart_view_period_description",
       "en": "Configure history diapason to be shown on chart from 1 to 72 hours",
-      "fi": "",
+      "fi": "Määritä historian tarkastelujakso 1 - 72 tuntia",
       "ru": "Установите диапазон для отображения на графиках от 1-го до 72-х часов",
       "sv": ""
     },
     {
       "ident_ios": "",
       "ident_android": "settings_gateway",
       "en": "Gateway Settings.",
-      "fi": "",
+      "fi": "Välityspalvelimen asetukset",
       "ru": "Настройки шлюза",
       "sv": ""
     },
     {
       "ident_ios": "",
       "ident_android": "settings_gateway_details",
       "en": "When you add a URL here, the app will execute an HTTP POST with the linked sensors when it does background scanning. \n\nFor more info, check the documentation.",
-      "fi": "",
+      "fi": "Sovellus lähettää anturitiedot HTTP POST -toiminnolla määritettyyn välityspalvelinosoitteeseen taustaskannauksen yhteydessä. \n\nLue lisää ohjesivulta.",
       "ru": "Если вы укажете URL здесь, приложение будет выполнять HTTP POST запрос для привязанных датчиков.\n\nИзучите документацию",
       "sv": ""
     },
     {
       "ident_ios": "",
       "ident_android": "settings_gateway_details_link",
       "en": "check the documentation.",
-      "fi": "",
+      "fi": "ohjesivu",
```

Comment:
it should be part of "settings_gateway_details". "ohjesivulta" or "lisää ohjesivulta"